var/spack/repos/builtin/packages/mepo/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Mepo(PythonPackage):
```

Comment:
Have you reviewed the naming convention when naming the package (https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name)?

Reply:
Well, in the main comment I said:

> Note: `mepo` is more like `awscli` in that it is a commandline utility written in Python rather than a Python library. So following the [Spack Python packaging name advice](https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name) we do not prefix it with `py-`

`mepo` is a code that is written in Python, but we see no need to ever expose the `mepo` internals a la `import mepo`. So, I figured like how `awscli` has:

https://github.com/spack/spack/blob/4d604c8c9f12e41a2fdb0292e40b35e19c64f3a5/var/spack/repos/builtin/packages/awscli/package.py#L10

I use:
```python
class Mepo(PythonPackage):
```

Is there a different way to do this?

Reply:
🤦 Missed the note.